README.md

```diff
@@ -7,129 +7,43 @@
 
 # OnDemand Loop
 
-Application to transfer data from heterogeneous research data repositories into Open OnDemand. With the possibility of syncing the data back.
+Welcome to the OnDemand Loop documentation.
 
-Creating the perfect loop for research data
+[**OnDemand Loop**](https://github.com/IQSS/ondemand-loop) is a companion application to [**Open OnDemand**](https://openondemand.org), designed to streamline the movement of research data between high-performance computing (HPC) clusters and remote repositories such as [**Dataverse**](https://dataverse.org), or [**Zenodo**](https://zenodo.org).
 
-The first remote repository integration is Dataverse.
-Dataverse will be used as the reference implementation for all features related to this application.
+The core goal of OnDemand Loop is to **lower the barrier for non-technical users** to interact with research data repositories. Following the Open OnDemand philosophy, it aims to provide a user-friendly interface for tasks that typically require complex command-line operations or custom scripts. Researchers can upload and download datasets to and from remote repositories directly from their HPC environment with minimal friction.
 
-OnDemand Loop has been designed from the ground up to be a companion to the Open OnDemand application.
-It is deployed into the Open OnDemand environment as another passenger app.
-It has been configured to be deployed under the folder: /var/www/ood/apps/sys/loop
+!!! warning "Beta Notice:"
 
-The OnDemand environment will launch the application in the same way as the Dashboard app.
-The URL will be https://<ood-server>/pun/sys/loop
+    *OnDemand Loop* is currently in **Beta** status. While we strive to provide a stable experience, please be aware of the following:
+
+    - You may encounter occasional bugs or incomplete features.
+    - User interface and workflow changes may occur without backward compatibility guarantees.
+    - Minor UI/UX inconsistencies are expected as the product evolves.
+    - We welcome feedback and bug reports to help us improve!
```

Comment:
This warning is not visualized with good formatting in GitHub, unlike in the Guide markdown. Could you change the formatting? This is how it is seen from my browser: 

<img width="992" height="246" alt="image" src="https://github.com/user-attachments/assets/63af5e82-4230-4b34-a08d-9a8f6dd6d0ed" />